src/ui/components/AccountSelector.js

```diff
@@ -1,47 +1,39 @@
 // @flow
-
 import React from "react";
 import {createMuiTheme, ThemeProvider} from "@material-ui/core/styles";
 import {TextField} from "@material-ui/core";
-import {format} from "../../ledger/grain";
 import {Autocomplete} from "@material-ui/lab";
 import {Ledger, type Account} from "../../ledger/ledger";
 
 type DropdownProps = {|
   +ledger: Ledger,
-  +setCurrentIdentity: (Account | null) => void,
+  +setCurrentAccount: (Account | null) => void,
   +placeholder?: string,
 |};
 
 const theme = createMuiTheme({
   overrides: {
-    // Name of the rule
     text: {
-      // Some CSS
       color: "black",
     },
   },
 });
 
 export default function AccountDropdown({
   placeholder,
-  setCurrentIdentity,
+  setCurrentAccount,
   ledger,
 }: DropdownProps) {
   const items = ledger.accounts().filter((a) => a.active);
 
-  const onComboChange = (e, inputObj) => setCurrentIdentity(inputObj);
-
   return (
     <ThemeProvider theme={theme}>
       <Autocomplete
-        onChange={onComboChange}
+        onChange={(e, inputObj) => setCurrentAccount(inputObj)}
         fullWidth
         options={items}
-        getOptionLabel={(item) =>
-          `${item.identity.name} (${format(item.balance, 2)})`
-        }
-        style={{margin: "20px"}}
+        getOptionLabel={(item: Account) => item.identity.name}
+        style={{margin: "20px", marginTop: "-16px"}}
```

Comment:
Why the negative top margin?

Reply:
probably sufficient for this top margin to just be zero.